docs/how-to-deploy-cp4waiops-customsize.md

```diff
@@ -0,0 +1,353 @@
+
+1. (Optional) Verify the validity of the key by logging in to the IBM Entitled Registry using a container tool:  
+   Depending on what contianer system you are using, you might need to use `docker login` instead of `podman login` for following commands.
+
+   ```sh
+   export IBM_ENTITLEMENT_KEY=the key from the previous steps
+   podman login cp.icr.io --username cp --password "${IBM_ENTITLEMENT_KEY:?}"
+   ```
+
+### Update the OCP global pull secret
+
+[Update the OCP global pull secret](https://docs.openshift.com/container-platform/4.7/openshift_images/managing_images/using-image-pull-secrets.html) with the entitlement key.
+
+Keep in mind that the registry user for that secret is `cp`. A common mistakes is to assume the registry user is the name or email of the user owning the entitlement key.
+
+#### Update the global pull secret using the OpenShift console
+
+1. Navigate to the "Workloads > Secrets" page in the "Administrator" perspective.
+
+1. Select the object "pull-secret".
+
+1. Click on "Actions -> Edit secret".
+
+1. Scroll to the bottom of that page and click on "Add credentials", using the following values for each field:
+
+   - "Registry Server Address" `cp.icr.io`
+   - "Username": `cp`
+   - "Password": paste the entitlement key you copied from the [Obtain an entitlement key](#obtain-an-entitlement-key) setp
+   - "Email": any email, valid or not, will work. This fields is mostly a hint to other people who may see the entry in the configuration
+
+1. Click on "Save"
+
+
+
+## Install custom sizing CP4WAIOps from UI
+
+### Login to Argo CD
+
+You can now login to Argo CD UI as follows by clicking the drop down menu on top right.
+
+![w](images/gitops-menu.png)
+
+Argo CD UI will be popped up and you can login using `LOG IN VIA OPENSHIFT`.
+
+![w](images/gitops-login.png)
+
+
+### Grant Argo CD Cluster Admin Permission
+
+From Red Hat OpenShift Console, go to `User Management` > `RoleBindings` > `Create binding`. Use the form view to configure the properties for the `ClusterRoleBinding` with values as follows, and click the `Create` button.
+
+- Binding type
+  - Cluster-wide role binding (ClusterRoleBinding)
+- RoleBinding
+  - Name: argocd-admin
+- Role
+  - Role Name: cluster-admin
+- Subject
+  - ServiceAccount: check it
+  - Subject namespace: openshift-gitops
+  - Subject name: openshift-gitops-argocd-application-controller
+
+
+
+
+### Install custom sizing CP4WAIOps Using All-in-One Configuration
+
+
+#### Install AI Manager and Event Manager in One Go
+
+The all-in-one configuration allows you to install following components in one go:
+
+- Ceph storage (optional)
+- AI Manager
+- Event Manager
+
+For AI Manager, you can specify the installation profile or sub-profile of small with custom sizing, including:
+
+- large
+- small
+- extra small: sub-profile requesting much less cpu and memory, only for demo, PoC, or dev environment in a resource (cpu and memory) limited cluster.
+  - x-small
```

Comment:
May need to explain what these profiles mean, e.g.: 
- x-small can be used to run LAD
- x-small-idle is the minimum profile in order to launch CP4WAIOps, just all pods up and running.
- x-small-custom is the a special profile that allows users to customize by themselves but the default values are based on `x-small` profile.